.github/workflows/chainloop_github_release.yml

```diff
@@ -0,0 +1,109 @@
+on:
+  workflow_call:
+    inputs:
+      project:
+        description: "The project this workflow belongs to. default: repository name"
+        type: string
+      workflow_name:
+        description: "The workflow name. default: parent workflow filename"
+        type: string
+      additional_materials:
+        description: "Additional materials to be attested. Commas separated list of values"
+        type: string
+    secrets:
+      api_token:
+        description: "Reference: https://docs.chainloop.dev/reference/operator/api-tokens#api-tokens"
+        required: true
+      cosign_key:
+        description: "The private key used to sign the attestation"
+        required: true
+      cosign_password:
+        description: "The password for the private key used to sign the attestation"
+        required: true
+
+jobs:
+  onboard_workflow:
```

Comment:
add guard 

```
if: github.ref_type == 'tag' # Guard to make sure we are releasing once
```

Reply:
Can we detect when it's an event of kind release or do we need to pass it?

Reply:
Yes, I've just added a guard that checks if the event that triggered the workflow is a release.